### PR TITLE
Add Enclaves DEX events

### PR DESCRIPTION
**Pull request by adamdossa**
Used the Contract Parser: https://contract-parser.d5.ai

### CODE REVIEW

**Comment by adamdossa** [comment]
Tagging @evgeniuz for review please.

File: dags/resources/stages/parse/table_definitions/enclaves_dex/EnclavesDEX_event_Cancel.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                {
+                    "indexed": false,
+                    "name": "s",
+                    "type": "bytes32"
+                }
+            ],
+            "name": "Cancel",
+            "type": "event"
+        },
+        "contract_address": "0xbF45F4280Cfbe7C2D2515A7D984B8c71c15E82B7",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/enclaves_dex/EnclavesDEX_event_Cancel.json:60]: Please lower case addresses

Comment (adamdossa) [reply dags/resources/stages/parse/table_definitions/enclaves_dex/EnclavesDEX_event_Cancel.json]: Done thanks @medvedev1088

**Comment by medvedev1088** [comment]
@adamdossa it should be available now

**Pull request merged**